backend/service/topology/cache_test.go

```diff
@@ -0,0 +1,20 @@
+package topology
+
+import (
+	"context"
+	"testing"
+
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTryAdvisoryLock(t *testing.T) {
```

Comment:
Im still not entirely sure how to test advisory locks with sqlmock, but still looking at it.